### ISSUE TITLE
Extra space at the bottom of the page

### ISSUE THREAD

**Issue by menigit** [pagedjs/pagedjs#182]
I have a long text that spreads over several pages, at the bottom of each page there is a large space that remains empty. The text does not fill the entire page, I tried to debug with the DevTools but found nothing, it seems that the Chunker allocates less content to each page than it can hold.
Has anyone encountered this and know how to deal with it?

I am using Paged.js file with:
```
let paged = new window.Paged.Previewer();
paged.preview(htmlContentString, ['style.css'], renderTo)
```
![image](https://github.com/pagedjs/pagedjs/assets/74985102/aab4da69-e0ba-4e26-a352-1ff0aaf65344)

**Comment by menigit** [comment]
Update: I found the problem! 

I was rendered the pages into my DOM and then copied the result (tags and styles) into a new html file to generate the pdf from it.

The problem is that I did not explicitly specify the font-family, so during the render process to the DOM it used the default font there (Ariel) but when I copied it to another file it used a different default font (Times New Roman).

In short, the solution: explicitly add the requested font in your CSS.
